### ISSUE TITLE
[Tabs] cacheType doesn't work

### ISSUE THREAD

**Issue by MyTheValentinus** [Semantic-Org/Semantic-UI#5605]
Hello,

I have multiples AJAX tabs, my server send html form to put in to the tab, but when i switch tab to another the form was reset, the html will be reset.

Have your idea of problem ?

Here is my options code to initialize tabs:
```javascript
{
      cache: true,
      auto: true,
      cacheType: "html",
      path: "/ressources/ajax/generate-form.ajax.php?tab="
}
```

And my html:

```html
<div class="content">
    <div class="ui top attached tabular menu" id="commands">
        <a class="item active" data-tab="commands*config.json">config.json</a>
        <a class="item" data-tab="commands*commands.json">commands.json</a>
    </div>
    <div class="ui bottom attached tab segment active" data-tabwithoutextension="commandsconfig"
         data-tab="commands*config.json">
        <form id="commandsconfig">
            <input type="text" name="test">
            <input type="text" name="test2">
        </form>
    </div>
    <div class="ui bottom attached tab segment" data-tabwithoutextension="commandscommands"
         data-tab="commands*commands.json">
        <form id="commandscommands">
            <input type="text" name="test">
            <input type="text" name="test2">
        </form>
    </div>
</div>
```

**Comment by MyTheValentinus** [comment]
Any update ? No label ?

**Comment by Banandrew** [comment]
Hi @MyTheValentinus, sorry for the delay, but everyone seems to be busy, and I barely worked with the tab component, so I’ll have to read the source to figure this out; I’ll try to come back to you with an answer this weekend.

**Comment by MyTheValentinus** [comment]
@Banandrew Ok no problem thanks ! I'm waiting your reply. Have good day
